packages/bpk-component-radio/src/form-test.js

```diff
@@ -0,0 +1,95 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow strict */
+
+import { useEffect } from 'react';
+
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import '@testing-library/jest-dom';
+
+import BpkRadio from './BpkRadio';
+
+describe('BpkRadio form test', () => {
+  it('should work as a form component in a form', async () => {
+    render(
+      <form data-testid="form">
+        <BpkRadio type="radio" name="radio" data-testid="myradio" />
+        <button type="submit">Submit</button>
```

Comment:
Are the submit buttons just to make it closer to a real use case?

Reply:
Yeah basically they don't have any functionality in the test but just to make it look like a normal form.